### DISCUSSION TITLE
Upnp don"t work

### DISCUSSION THREAD

**Discussion by knkn59** [rern/rAudio#887] [Bug Reports]
taskset[252]: :2:src/mpdcli.cxx:113::MPDCli::openconn: mpd_connection_new failed: Connection refused
taskset[252]: :2:src/main.cxx:640::MPD connection failed
taskset[252]: :2:src/mpdcli.cxx:113::MPDCli::openconn: mpd_connection_new failed: Connection refused
taskset[252]: :2:src/main.cxx:640::MPD connection failed

**Comment by marcelgsteiger** [comment]
...same problem here, with current version 20220917

**Reply by rern** [reply]
https://github.com/rern/rAudio-1/discussions/1031#discussioncomment-3681366
